presentation/src/main/java/com/anytypeio/anytype/presentation/sets/ViewerEditWidgetUi.kt

```diff
@@ -0,0 +1,95 @@
+package com.anytypeio.anytype.presentation.sets
+
+import com.anytypeio.anytype.core_models.DVViewer
+import com.anytypeio.anytype.core_models.DVViewerType
+import com.anytypeio.anytype.core_models.Id
+import com.anytypeio.anytype.core_models.ObjectTypeIds
+import com.anytypeio.anytype.core_models.ObjectWrapper
+import com.anytypeio.anytype.domain.objects.StoreOfObjectTypes
+import com.anytypeio.anytype.domain.objects.StoreOfRelations
+
+data class ViewerEditWidgetUi(
+    val showWidget: Boolean,
+    val id: Id? = null,
```

Comment:
what is id here, if it is nullable?

Reply:
ViewerEditWidgetUi is a state for all settings in any View. If id is empty, then the menu is not valid and is most likely hidden